Modules/Segmentation/LevelSets/include/itkThresholdSegmentationLevelSetFunction.hxx

```diff
@@ -21,7 +21,6 @@
 #include "itkImageRegionIterator.h"
 #include "itkGradientAnisotropicDiffusionImageFilter.h"
 #include "itkLaplacianImageFilter.h"
-#include "itkImageFileWriter.h"
```

Comment:
This `#include "itkImageFileWriter.h"` appears to be introduced with commit 04f04d55d39a1e6950d7f0e12e2b689ad4cbbaed "ENH: Filter now incorporates an optional Laplacian term...", Josh Cates, July 4, 2003: https://github.com/InsightSoftwareConsortium/ITK/blob/04f04d55d39a1e6950d7f0e12e2b689ad4cbbaed/Code/Algorithms/itkThresholdSegmentationLevelSetFunction.txx